doc/user/content/release-notes.md

```diff
@@ -107,80 +107,98 @@ These changes are present in [unstable builds](/versions/#unstable-builds) and
 are slated for inclusion in the next stable release. There may be additional
 changes that have not yet been documented.
 
+- **Breaking change.** Return an empty list for slice operations that retrieve
+  no elements (e.g. the beginning of the slice's range exceeds the length of the
+  list); previously Materialize returned NULL.
+
+- **Breaking change.** Decrease minimum [`interval`](/sql/types/interval) value
+  from '-2147483647 months -2147483647 days -2147483647 hours -59 minutes
+  -59.999999 seconds' to '-2147483648 months -2147483648 days -2147483648 hours
+  -59 minutes -59.999999 seconds' to match PostgreSQL's behavior {{% gh 10598 %}}.
+
+- Fix a bug where too many columns were returned when both `*` and a
+  table function appeared in the `SELECT` list {{% gh 10363 %}}.
+
+{{< comment >}}
+Only add new release notes above this line.
+
+The presence of this comment ensures that PRs that are alive across a release
+boundary don't silently merge their release notes into the wrong place.
+{{</ comment >}}
+
+{{% version-header v0.20.0 %}}
+
+- **Breaking change.** In Kafka sources and sinks, do not default the SSL
+  parameters for the Confluent Schema Registry to the SSL parameters for the
+  Kafka broker.
+
+  SSL parameters for the Confluent Schema Registry must now always be provided
+  explicitly, even when they are identical to the SSL parameters for the Kafka
```

Comment:
> even when they are identical to the SSL parameters for the Kafka broker

Ah, had a hunch that this was contradicting the docs, but just double checked this seems to have changed there too. That's too bad, it makes the syntax a bit verbose.

Reply:
Yep, totally, some discussion on that point here: https://github.com/MaterializeInc/materialize/pull/10381#issuecomment-1029540740. The tl;dr is we have a plan for removing all of the duplication from `CREATE SOURCE`, so didn't want to spend time removing just some of the duplication.

I'm working on a design doc for this `CREATE CONNECTOR` (see also https://github.com/MaterializeInc/materialize/issues/8372; apparently I've been talking about this for a while 🙈 ).

Reply:
Oh cool, I did not know about `CREATE CONNECTOR`!